clusterloader2/testing/request-benchmark/config.yaml

```diff
@@ -0,0 +1,72 @@
+# Request benchmark
+{{$configMapBytes := DefaultParam .CL2_BENCHMARK_CONFIG_MAP_BYTES 1000}}
+{{$configMapGroup := DefaultParam .CL2_BENCHMARK_CONFIG_MAP_GROUP "benchmark-config-map"}}
+{{$configMapNumber := DefaultParam .CL2_BENCHMARK_CONFIG_MAP_NUMBER 1}}
+{{$inflight := DefaultParam .CL2_BENCHMARK_INFLIGHT 10}}
+{{$pods := DefaultParam .CL2_BENCHMARK_PODS 1}}
+# URI example: /api/v1/namespaces/%namespace%/pods
+{{$uri := DefaultParam .CL2_BENCHMARK_URI ""}}
+
+name: Request benchmark
+namespace:
+  number: 1
+tuningSets:
+- name: Sequence
+  parallelismLimitedLoad:
+    parallelismLimit: 1
+steps:
+- name: Setup permissions
+  phases:
+  - namespaceRange:
+      min: 1
+      max: 1
+    tuningSet: Sequence
+    replicasPerNamespace: 1
+    objectBundle:
+    - basename: benchmark-role
+      objectTemplatePath: cluster-role.yaml
+  - namespaceRange:
+      min: 1
+      max: 1
+    tuningSet: Sequence
+    replicasPerNamespace: 1
+    objectBundle:
+    - basename: benchmark-role-binding
+      objectTemplatePath: cluster-role-binding.yaml
+- name: Create config map(s)
+  phases:
+  - namespaceRange:
+      min: 1
+      max: 1
+    tuningSet: Sequence
+    replicasPerNamespace: {{$configMapNumber}}
+    objectBundle:
+    - basename: {{$configMapGroup}}
+      objectTemplatePath: configmap.yaml
+      params:
+        bytes: {{$configMapBytes}}
+        group: {{$configMapGroup}}
+- module:
+    path: modules/benchmark-deployment.yaml
+    params:
+      inflight: {{$inflight}}
```

Comment:
Nit: reorder fields? "pods" is a different category (how many pods) than "uri" and "inflight" (what each pod should be doing)?. Also maybe some other name to make it more clear? "replicas"?

Reply:
+1 to "replicas" as it's the name used later in the spec.